public/igameevents.h

```diff
@@ -22,7 +22,7 @@
 class CMsgSource1LegacyGameEvent;
 class CPlayerSlot;
 class CBasePlayer;
-class CEntityIndex;
+struct CEntityIndex;

```

Comment:
``CEntityIndex`` is a class https://github.com/alliedmodders/hl2sdk/blob/22c1807de632a50fbade7e8035c50f893cd2e108/public/eiface.h#L114